test/livebook/live_markdown/markdown_helpers_test.exs

```diff
@@ -9,6 +9,16 @@ defmodule Livebook.LiveMarkdown.MarkdownHelpersTest do
       assert markdown == reformat(markdown)
     end
 
+    test "other emphasis" do
+      markdown = "The Game, _Mrs Hudson_, is on!"
+      assert markdown == reformat(markdown)
+    end
+
+    test "nested emphasis" do
+      markdown = "The *Game, _Mrs Hudson_, is* on!"
+      assert markdown == reformat(markdown)
```

Comment:
This test fails BADLY because the `_` are converted to `*` in a way that changes the meaning. 

This is an example of "formatting not stable under repeat formatting"; the first time that the Markdown above is formatted it produces `The *Game, *Mrs Hudson*, is* on!`; the second formatting discards the space after the first comma.

Reply:
The Gruber Markdown dingus and GFM don't agree about what this SHOULD be rendered as, FWIW.

* Gruber Markdown thinks this should display as `<p>The <em>Game, _Mrs Hudson_, is</em> on!</p>`

* GFM thinks this should display as `<p>The <em>Game, <em>Mrs Hudson</em>, is</em> on!</p>`